styles/components/nav.css

```diff
@@ -0,0 +1,21 @@
+.grid__nav {
+  background-color: rgb(64, 192, 224);
+  display: grid;
+  grid-template-columns: repeat(4, 25%);
+  border-top: 1px solid black;
+}
+
+.nav__a {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+nav > a:not(:last-child) {
```

Comment:
i think in the css files we should just use classes. on the other hand i also didnt figure out where to place those😅

Reply:
Oh yes I  fixed it. Just place the rule into a class and give this class the first three nav items.